redash/handlers/query_results.py

```diff
@@ -89,8 +43,8 @@ def run_query(query, parameters, data_source, query_id, max_age=0):
     if query_result:
         return {'query_result': query_result.to_dict()}
     else:
-        job = enqueue_query(query.text, data_source, current_user.id, metadata={
-            "Username": current_user.email,
+        job = enqueue_query(query.text, data_source, current_user.id, current_user.is_api_user(), metadata={
+            "Username": repr(current_user) if current_user.is_api_user() else current_user.email,
```

Comment:
What `repr(current_user)` returns? As this is used in logging we need to make sure we don't pass the actual API key there.

Reply:
It's [indeed the API key](https://github.com/getredash/redash/blob/master/redash/models/users.py#L375). What other way do suggest we identify an api user otherwise in the log?

Reply:
Actually it's not. It's either the object id in case of the `ApiKey` model:

https://github.com/getredash/redash/blob/ec4f77c8b7fbcb4b04821e98bc12a0f994a6b16e/redash/models/users.py#L375

(notice it's `api_key.id` and not `api_key.api_key`)

... or the query id in case of a query API key:

https://github.com/getredash/redash/blob/ec4f77c8b7fbcb4b04821e98bc12a0f994a6b16e/redash/authentication/__init__.py#L137

So all good here.